test-integration/test_integration/fixtures/failing-project-after-output/predict.py

```diff
@@ -0,0 +1,12 @@
+import time
```

Comment:
Maybe it doesn't matter, but this project name `failing-project-after-output` differs from all the others which end with `-project`. This could be `failing-after-output-project` for consistency.

Reply:
Good catch, thank you! Fixed in 6a4551b